templates/index.html

```diff
@@ -63,15 +63,12 @@ <h2>Quick start</h2>
   </div>
   <div class="row">
     <div class="col-6">
-      <p>The recommended method of including Vanilla Framework in your project is via <a href="https://www.npmjs.com/package/vanilla-framework">npm</a>. You can then simply include the framework in your SCSS files and compile.</p>
+      <p>The recommended method of including Vanilla Framework in your project is via <a href="https://www.npmjs.com/package/vanilla-framework">npm</a> or yarn. You can then simply include the framework in your SCSS files and compile.</p>
       <p>For other methods, please see the <a href="/docs">advanced usage docs.</a></p>
     </div>
     <div class="col-6">
-      <pre><code>npm install --save-dev vanilla-framework
-<span class="token builtin">export</span> SASS_PATH=`<span class="token builtin">pwd</span>`/node_modules:<span class="token variable">${SASS_PATH}</span>
-
-<span class="token comment">// Add to your main build scss file the following line</span>
-<span class="token keyword">@import</span> <span class="token string">'vanilla-framework'</span></code></pre>
+      <pre><code>yarn add sass vanilla-framework</code></pre>
+      <pre><code>@import "vanilla-framework";<br>@include vanilla;</code></pre>
```

Comment:
Could you keep the syntax coloring tokens? We don't want to run prism on whole site, but we can at least hardcode some colors here:

```
<pre><code><span class="token keyword">@import</span> <span class="token string">"vanilla-framework"</span><span class="token punctuation">;</span>
<span class="token keyword">@include</span> vanilla<span class="token punctuation">;</span></code></pre>
```